components/Navbar.tsx

```diff
@@ -17,7 +17,7 @@ export default function Navbar() {
             <Link
               key={title}
               className={`transition-colors hover:border-cesium-900 hover:text-cesium-900`}
-              href={`/${title.toLowerCase()}`}
+              href={`/${title.toLowerCase().replace(/\s+/g, "_")}`}
```

Comment:
```suggestion
              href={`/${title.replace(/\s+/g, "")}`}
```

This needs to be updated to match the naming convention above.